assets/js/inpagelink.js

```diff
@@ -1,32 +1,35 @@
 import {forEach} from 'lodash'
 import domready from './domready'
 
-export const classTrigger = 'js-inpagelink'
+export const classTrigger = 'js-inpagelink';
 
 export default function() {
-  return inPageLink()
+  return inPageLink();
 }
 
 export function inPageLink() {
-  const nodeList = document.getElementsByClassName(classTrigger)
-  forEach(nodeList, applyInPageLink)
+  const nodeList = document.getElementsByClassName(classTrigger);
```

Comment:
The lodash dependency can be removed by casting the node list to an array like so which will allow you to use native forEach

```suggestion
  const nodeList = [...document.getElementsByClassName(classTrigger)];
```

Reply:
Not related to changes in the context of this PR, create an issue?